src/index.tsx

```diff
@@ -16,17 +16,18 @@ import Preonboarding from './pages/Preonboarding';
 import Onboarding from './pages/Onboarding';
 import Landing from './pages/Landing';
 import history from './context/History';
-import Verifiers from './pages/Verifiers';
 import LogExplorer from './pages/LogExplorer';
 import './fonts/SuisseIntl-Regular.woff'
-import Miners from './pages/Miners';
-import Title from './components/Title';
 import ClientDetails from './pages/ClientDetails'
 import { CookiesProvider } from 'react-cookie';
 import * as Sentry from "@sentry/react";
 import { Integrations } from "@sentry/tracing";
 import Footer from './components/Footer';
 import LdnApplication from './pages/LdnApplication';
+import Verifiers2 from './pages/Verifiers2';
```

Comment:
@huseyincansoylu  change this with Verifiers, you should replace the old component, not adding a new one